Include/cpython/unicodeobject.h

```diff
@@ -1090,15 +1096,15 @@ PyAPI_FUNC(int) _PyUnicode_IsLinebreak(
     const Py_UCS4 ch         /* Unicode character */
     );
 
-/* Py_DEPRECATED(3.3) */ PyAPI_FUNC(Py_UCS4) _PyUnicode_ToLowercase(
+PyAPI_FUNC(Py_UCS4) _PyUnicode_ToLowercase(
```

Comment:
It's kind of surprising to **add** deprecation and **remove** deprecation in the same PR. Would you mind to remove deprecation in a separated PR?

Reply:
Agreed, doing both at once makes it a bit harder to differentiate. Especially when looking back at the PR from a later date.